.vscodeignore

```diff
@@ -20,5 +20,6 @@ gulp*
 webpack.config*
```

Comment:
While you're working on this file - what are your thoughts on removing `package-lock.json` as well? There's been a few times where I go back inside an old vsix to see what version of a dependency is used, but I won't be able to do that going forward since they all get webpacked together. Seems including the single json file won't be too big to ship in the vsix, right? Or is there a better way to get this info?

Reply:
Sounds like a good idea regardless, thanks.